### ISSUE TITLE
make a decent command line tool

### ISSUE THREAD

**Issue by yossigil** [SpartanRefactoring/spartan-refactoring#125]
start at Application

**Comment by yossigil** [comment]
News?

**Comment by M4tt30rru** [comment]
Working on it. I supposed that for today the priority was the other topic since we had a meeting in the afternoon, so I focused more on that.

**Comment by yossigil** [comment]
We need this for experiments. @danielmittelman  and @OriRoth  might be able to help.

**Comment by OriRoth** [comment]
Notice new features in `plugin.revision` package, especially `Applicator`/`EventApplicator`/`SpartanizationHandler` classes.
@yossigil @M4tt30rru does someone works on this issue? I strongly believe the tools I have developed can be used here... they are quite powerful.

**Comment by yossigil** [comment]
You are right! They should. Lets take it (one day at a time). We do work on our tools: there is an IneractiveSpartanizer class and a BatchSpartanizer. See if you can integrate.

**Comment by M4tt30rru** [comment]
I can work on this today.

**Comment by yossigil** [comment]
@ORIMARCO: FYI.
